components/BB.js

```diff
@@ -251,15 +259,19 @@ export class BB extends Component {
               <Grid item lg={8} md={8} sm={7} xs={12}>
                 <Grid container spacing={16}>
                   <Grid item xs={12} md={6}>
+                    <InputLabel className={classes.sortByLabel}>
```

Comment:
will screen readers associate this label with the Select?

Reply:
Probably not

Reply:
eh, it passes Axe tests.

Reply:
Adding a `for` to the label